monai/handlers/checkpoint_loader.py

```diff
@@ -80,5 +80,14 @@ def __call__(self, engine: Engine) -> None:
         """
         checkpoint = torch.load(self.load_path, map_location=self.map_location)
 
+        # save current max epochs setting in the engine, don't overwrite it if larger than max_epochs in checkpoint
+        prior_max_epochs = engine.state.max_epochs
         Checkpoint.load_objects(to_load=self.load_dict, checkpoint=checkpoint)
+        if engine.state.epoch > prior_max_epochs:
+            raise ValueError(f"Epoch count ({engine.state.epoch}) in checkpoint is larger than \
```

Comment:
I think this will end up printing the message across different lines with the space included. The line should be broken up with multiple string literals to avoid this:

```python
raise ValueError(f"Epoch count ({engine.state.epoch}) in checkpoint is larger than "
               f"the `engine.state.max_epochs` ({prior_max_epochs}) of engine. To further train from checkpoint, "
                "construct trainer with `max_epochs` larger than checkpoint's epoch count. "
                "To use checkpoint for inference, no need to load state_dict for the engine.")
```

Reply:
sounds good.